test/endtoend/providers/main.js

```diff
@@ -8,7 +8,7 @@ const Dropbox = require('@uppy/dropbox')
 const Tus = require('@uppy/tus')
 
 const isOnTravis = !!(process.env.TRAVIS && process.env.CI)
-const companionUrl = isOnTravis ? 'http://companion.test:3030' : 'http://localhost:3030'
+const companionUrl = isOnTravis ? 'http://companion.test:3030' : 'http://localhost:3020'
```

Comment:
wdio runs Companion on :3030. can't we use that here?

Reply:
I resorted to 3020 for the following reasons:

1. Oauth Redirect URI for the Providers in dev env is currently set to allow only http://localhost:3020
2. I thought since this is meant to run locally there's no need to duplicate the usage of secrets nor is there a need to bother running a different companion instance.

I think the companion at 3030 serves for url-plugin on travis which does not need Oauth dance, also it made sense to have it because the other providers were intended to run on travis, but unfortunately, that's not the case anymore

Reply:
@goto-bus-stop any reservations on this?